x/ibc/02-client/keeper/querier.go

```diff
@@ -75,3 +75,26 @@ func QuerierVerifiedRoot(ctx sdk.Context, req abci.RequestQuery, k Keeper) ([]by
 
 	return bz, nil
 }
+
+// QuerierCommitter defines the sdk.Querier to query a committer
+func QuerierCommitter(ctx sdk.Context, req abci.RequestQuery, k Keeper) ([]byte, sdk.Error) {
```

Comment:
Note: these are not required atm since we're querying the store directly. I guess it's fine to have them tho